Source/items.cpp

```diff
@@ -4629,8 +4608,8 @@ void SpawnBoy(int lvl)
 		Items[0]._iSeed = AdvanceRndSeed();

```

Comment:
There are no early returns from this point in the function, `boyitem` can be used instead of `Items[0]` without side effects.